tests/filecheck/transforms/dispatch_elementwise_mult.mlir

```diff
@@ -0,0 +1,23 @@
+// RUN: ./compiler/snax-opt %s -p dispatch-elementwise-mult --allow-unregistered-dialect --print-op-generic | filecheck %s
```

Comment:
Is this how that works? I'm not sure if we need the GENERIC roundtrip here instead maybe.
```suggestion
// RUN: XDSL_ROUNDTRIP
```

Reply:
also added a roundtrip test